torchdynamo/debug_utils.py

```diff
@@ -142,11 +152,18 @@ def isolate_inductor_fails(fx_g, args, env=None):
     return False
 
 
-def inductor_fails(fx_g, args, check_str=None):
+def inductor_fails(fx_g, args, check_str="CompilationError"):
     from torchinductor import config
     from torchinductor.compile_fx import compile_fx_inner
 
-    config.autotune = False
+    config.triton.autotune = False
+
+    try:
+        result = fx_g(*args)
+        assert isinstance(result, (tuple, list))
+        assert not any([isinstance(x, (tuple, list)) for x in result])
```

Comment:
What about sets maps or other collection objects? Maybe instead we can flatten and check that the size stays the same?

Reply:
This is mimicking the code in TorchInductor's lowering, which assumes the output is a list/tuple of tensors.https://github.com/pytorch/torchdynamo/blob/main/torchinductor/graph.py#L250